wifiphisher/extensions/deauth.py

```diff
@@ -211,11 +203,9 @@ def _add_clients(self, sender, receiver, bssid):
 
         # return the client and packets if valid and None otherwise
         # it uses short circuiting to improve performance
-        return (sender not in non_valid_addresses and
-                receiver not in non_valid_addresses and
-                (sender == bssid and (receiver, packets()) or
-                 receiver == bssid and (sender, packets())) or
-                None)
+        return (sender not in non_valid_addresses and receiver not in non_valid_addresses and
+                (sender == bssid and (receiver, packets()) or receiver == bssid and
+                 (sender, packets())) or None)
```

Comment:
This is an example where the code was more readable before `yapf`.

The number of columns is not standard. Sometimes it makes sense to limit them, other times you may want to use all 99 columns.

I believe we should revert back in cases like these.

Reply:
I have update and changed it back to the previous state and by using the `# yapf: disable` it will not modify it again. However it should be noted things like this should be really rare as the whole point using `yapf` was to forget about formatting 😄  .